app/views/events/show.html.erb

```diff
@@ -10,5 +10,9 @@
   <dd><%= @event.location %></dd>
 </dl>
 
-<%= link_to 'Edit', edit_event_path(@event) %>
-<%= render partial: 'delete-link', locals: { event: @event } %>
+<% if policy(@event).edit? %>
+  <%= link_to 'Edit', edit_event_path(@event) %>
+<% end %>
```

Comment:
`<%= link_to 'Edit', edit_event_path(@event) if policy(@event).edit? %>`

Could probably one line these. That one is 70 char.